src/app/shared/docs-components/anatomy-item.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-template #content><ng-content></ng-content></ng-template>
 
 <ng-container *ngIf="isOptional; else requiredStyle">
-  <div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">{{anatomyNumber}}</div><div class="anatomy-description"><ng-container *ngTemplateOutlet="content"></ng-container> <span class="sky-deemphasized">(optional)</span></div></div>
+  <div class="anatomy-item anatomy-item-optional"><div class="anatomy-marker">{{anatomyNumber}}</div><div class="anatomy-description"><ng-container *ngTemplateOutlet="content"></ng-container> <span class="sky-deemphasized"> (optional)</span></div></div>
```

Comment:
Just want to double check that we intended to add this space?

Reply:
This looks inadvertent to me, but I don't know. (If I made this change, it was definitely accidental. But I checked my commits, and I don't see this change there.)

Reply:
I removed the extra space.